### ISSUE TITLE
core-js not found

### ISSUE THREAD

**Issue by vladnicula** [istarkov/react-components-markdown#121]
Seems like this breaks with core-js not found when bundling with webpack. Maybe it's because core-js is only in dev dependencies :-?

<img width="1121" alt="screen shot 2016-11-09 at 10 46 00" src="https://cloud.githubusercontent.com/assets/126038/20132068/d0820f0a-a669-11e6-898e-b30d59a19c17.png">

Fixed it by adding core-js in my package json.

**Comment by istarkov** [comment]
Yes, will think to add notice about into readme, or to fix somehow else
